coderunners/app.py

```diff
@@ -7,29 +7,11 @@ def run_code_lambda(event, context):
     This lambda has no internet access and no permissions to access resources
     It is run on an isolated container after which the results are returned to the "caller" function
     """
-    from services import check_code
+    from coderunners.services import check_code
```

Comment:
Should we move this to global scope?

Reply:
No need for it yet.

Reply:
I think we should stick to the same project structure across the whole proejct. If we have `from bouncer.judges import check_equality` in the global scrope for the bouncer, we should probably have the same type of import in the coderunner as well